capstone/src/main/resources/static/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Events</title>
+    <link rel="stylesheet" href="style1.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <h1>Events</h1>
```

Comment:
I like how bare bones this is to start with. You can always enhance it and add more css and headers etc. later!